apps/wistia-videos/src/components/Field.tsx

```diff
@@ -31,37 +33,47 @@ const StyledImageContainer = styled.div`
 
 const Field = (props: FieldProps) => {
   const { sdk } = props;
-  const [data, updateData] = useState<WistiaItem[] | []>([]);
+  const [data, updateData] = useState<WistiaVideo[] | []>([]);
   const [selectedIds, setIds] = useState<number[]>([]);
-  const [dropdownData, filterDropdownData] = useState<WistiaItem[] | []>([]);
+  const [dropdownData, filterDropdownData] = useState<WistiaVideo[] | []>([]);
   const [loading, updateLoadingStatus] = useState(true);
+  const [end, setEnd] = useState(100);
```

Comment:
maybe make this a constant? Since i see this 100 used in other places.

Maye somethign like `VIDEOS_PER_PAGE`?